From 9c4a7545cb70acb09af610738d7b6b75e2db6f9e Mon Sep 17 00:00:00 2001
Message-Id: <patch-6.6-9c4a7545cb7-20210615T161330Z-avarab@gmail.com>
In-Reply-To: <cover-0.6-00000000000-20210615T161330Z-avarab@gmail.com>
References: <cover-0.6-00000000000-20210615T161330Z-avarab@gmail.com>
From: =?UTF-8?q?=C3=86var=20Arnfj=C3=B6r=C3=B0=20Bjarmason?=
 <avarab@gmail.com>
Date: Tue, 15 Jun 2021 18:07:03 +0200
Subject: [PATCH 6/6] pack-protocol doc: use "www-data" in place of "alice"
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Replace the example of "alice" in the pack-protocol.txt documentation
added in b31222cfb7f (Update packfile transfer protocol documentation,
2009-11-03) with "www-data". This is now consistent with the recently
changed examples in the tutorial and git-daemon documentation.

Signed-off-by: Ævar Arnfjörð Bjarmason <avarab@gmail.com>
---
 Documentation/technical/pack-protocol.txt | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/Documentation/technical/pack-protocol.txt b/Documentation/technical/pack-protocol.txt
index e13a2c064d1..0f43a290e59 100644
--- a/Documentation/technical/pack-protocol.txt
+++ b/Documentation/technical/pack-protocol.txt
@@ -136,10 +136,10 @@ directory, because the Git client will run:
 The exception is if a '~' is used, in which case
 we execute it without the leading '/'.
 
-      ssh://user@example.com/~alice/project.git,
+      ssh://user@example.com/~www-data/project.git,
 		     |
 		     v
-   ssh user@example.com "git-upload-pack '~alice/project.git'"
+   ssh user@example.com "git-upload-pack '~www-data/project.git'"
 
 Depending on the value of the `protocol.version` configuration variable,
 Git may attempt to send Extra Parameters as a colon-separated string in
-- 
2.32.0.555.g0268d380f7b

